packages/common/src/polyfill.ts

```diff
@@ -0,0 +1,20 @@
+import { shouldPolyfill as shouldPolyfillCanon } from '@formatjs/intl-getcanonicallocales/should-polyfill'
```

Comment:
I'm not sure if this is the best place for this polyfill.  I wanted it to be in mobile since its specific to mobile but the mobile code wasn't being run soon enough, it seems, to run before the logging started.

Reply:
I also added some doc strings to the this file for clarity.